code/modules/surgery/bodyparts/helpers.dm

```diff
@@ -13,7 +13,7 @@
 /mob/living/carbon/has_hand_for_held_index(i)
 	if(i)
 		var/obj/item/bodypart/L = hand_bodyparts[i]
-		if(L && !L.disabled)
+		if(L && (!L.disabled || HAS_TRAIT(src, TRAIT_NOLIMBDISABLE)))		// Wasp edit - Lavachem rebalance
```

Comment:
I don't think any of these checks are needed, since the trait is already checked for when the limb is initially being disabled.

Reply:
This is meant to get the limb working if it is already disabled.

Reply:
Looks like this might have been changed in the the mobility changes, doesn't look like it's needed now